### ISSUE TITLE
[Omnichannel] Thank you message not visible when registration form is enabled

### ISSUE THREAD

**Issue by qsolutions-pl** [RocketChat/Rocket.Chat#27903]
### Steps to reproduce:

1. Enable OmniChannel
2. Enable Registration for OC and make name and email required
3. Configure `Conversation Finished Message`

### Expected behavior:

Once user/agent finishes the chat he/she should see the `Conversation Finished Message` message

### Actual behavior:

The user does not see the `Conversation Finished Message`, a new registration form is presented
![image](https://user-images.githubusercontent.com/4256191/215743448-8697e8ae-0018-4b57-97a5-4f732ed56850.png)
I've recorded a video and the "thank you message" is visible but for like 0.1 seconds.

### Server Setup Information:

- Version of Rocket.Chat Server: 5.4.2
- Operating System: Ubuntu 20.04
- Deployment Method:  snap
- Number of Running Instances: 1
- NodeJS Version: v14.19.3
- MongoDB Version: `4.4.15`

### Client Setup Information

- Desktop App or Browser Version: all browsers
- Operating System: all systems

**Comment by dudanogueira** [comment]
Indeed, when the form registration is enabled, it will get back to the form without display the conversation ended message.

Reproduced this on latest 6.0.0-develop commit ea59e1da1

**Comment by durgesh077** [comment]
I want to work on this issue

**Comment by qsolutions-pl** [comment]
> I want to work on this issue . Please !! can you provide me with more details regarding it .

All required steps are described in the first message
